exonum-java-binding/common/src/main/java/com/exonum/binding/common/proofs/list/CheckedListProof.java

```diff
@@ -37,6 +37,7 @@
 public interface CheckedListProof<E> extends CheckedProof {
   /**
    * Get all list elements. There might be several consecutive ranges.
+   * Empty if the proof is a proof of absence.
```

Comment:
I would move it to `@return` statement and add there a short description when it's not empty